app/assets/stylesheets/nrl-styles.css.scss

```diff
@@ -98,10 +98,16 @@
       border: none;
   }
 }
- ////////////////////////////////
 
- .btn-custom {
-   background-color: #990000;
-   color: #fff;
-   margin-top: 2px;
- }
\ No newline at end of file
+////////////////////////////////
+
+.btn-custom {
+  background-color: #990000;
+  color: #fff;
+  margin-top: 2px;
+}
+
+.custom-button {
+  width: 10em;
```

Comment:
we should be consistent about using em / px / rm

Reply:
Yeah I agree with you. I am think of just using rem instead of px.